modules/requirements.txt

```diff
@@ -2,3 +2,4 @@ redis==2.10.5
 aiohttp==0.21.5
 wolframalpha==3.0
 pyquery==1.2.17
+python-twitter
```

Comment:
Exact version required. Any update could change the API of the library and module will be broken.